plugins/generator-1.17.1/forge-1.17.1/procedures/item_check_rarity.java.ftl

```diff
@@ -0,0 +1,2 @@
+<#include "mcitems.ftl">
+(${mappedMCItemToItemStackCode(input$item, 1)}.getRarity() == Rarity.${field$rarity})
```

Comment:
Rarity names changed so this needs a mapping system or additional if inside this template

Reply:
Rarity names didn't change, they're still `COMMON`, `UNCOMMON`, `RARE`, `EPIC`

Reply:
Oh, my bad, I was looking at enchantment rarities :D